src/components/RequesterPaysModal.ts

```diff
@@ -60,7 +71,9 @@ const RequesterPaysModal = ({ onDismiss, onSuccess }) => {
               {
                 disabled: !selectedGoogleProject,
                 onClick: () => {
-                  onSuccess(selectedGoogleProject);
+                  if (selectedGoogleProject) {
```

Comment:
Since we disable the button if `!selectedGoogleProject`, we don't really need to worry about that value not being set. However, we get a lint error without this.